Extension/test/unit/typeinfo.test.ts

```diff
@@ -0,0 +1,123 @@
+/* --------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation. All Rights Reserved.
+ * See 'LICENSE' in the project root for license information.
+ * ------------------------------------------------------------------------------------------ */
+
+import { describe, it } from 'mocha';
+import { ok, strictEqual } from 'node:assert';
+import { Async } from '../../src/Utility/Async/constructor';
+import { is } from '../../src/Utility/System/guards';
+import { hierarchy, typeOf } from '../../src/Utility/System/info';
+import { verbose } from '../../src/Utility/Text/streams';
+import { AnotherThree } from './examples/someclass';
+
+class Foo {
+
+}
+
+class Bar extends Foo {
+
+}
+
+class Baz extends Async(class Baz {
+
+})
+{
+
+}
+
+class Buzz extends Async(class Buzz extends Baz.class {
+
+})
+{
+
+}
+
+function foo() {
+
+}
+
+describe('Type Information', () => {
+    it('identify class of Async() type ', () => {
+        console.log(hierarchy(AnotherThree).join('::'));
+    });
+
+    it('is not failing subtest', () => {
+        ok(true, 'looks ok');
+        // ok(false,'some relevant assertion here');

```

Comment:
Is `// ok(false,'some relevant assertion here');` intentional, i.e. would uncommenting it be useful to verify the test fails? If so, can you add a comment that say so?

Reply:
leftover from a long time ago.